prover/crates/bin/prover_job_monitor/src/queue_reporter/witness_generator_queue_reporter.rs

```diff
@@ -0,0 +1,71 @@
+use async_trait::async_trait;
+use zksync_prover_dal::{Connection, Prover, ProverDal};
+use zksync_types::{
+    basic_fri_types::AggregationRound, protocol_version::ProtocolSemanticVersion,
+    prover_dal::JobCountStatistics,
+};
+
+use crate::{
+    metrics::{JobStatus, PROVER_JOB_MONITOR_METRICS},
+    task_wiring::Task,
+};
+
+/// `WitnessGeneratorQueueReporter` is a task that reports witness generator jobs status.
+/// Note: these values will be used for auto-scaling witness generators (Basic, Leaf, Node, Recursion Tip and Scheduler).
+#[derive(Debug)]
+pub struct WitnessGeneratorQueueReporter {}
```

Comment:
```suggestion
pub struct WitnessGeneratorQueueReporter;
```

not needed for empty struct.